src/Symfony/Component/Debug/Formatter/Formatter.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Debug\Formatter;
+
+use Symfony\Component\Debug\Exception\FlattenException;
+
+interface Formatter
+{
+    /**
+     * Sets the charset used by exception messages.
+     *
+     * @param string $charset The charset used by exception messages.
+     */
+    public function setCharset($string);
+
+    /**
+     * Gets the MIME type of the content returned by this formatter.
+     *
+     * @param string A MIME-type, possibly with a charset parameter.
```

Comment:
This method hasn't any argument. I think this is the description for the `@return` docblock.